src/textnote.ts

```diff
@@ -112,26 +71,33 @@ export class TextNote extends Note {
     super(noteStruct);
 
     this.text = noteStruct.text || '';
-    this.superscript = noteStruct.superscript;
-    this.subscript = noteStruct.subscript;
-    this.setFont(noteStruct.font);
+    if (noteStruct.glyph) {
+      this.text += TextNote.GLYPHS[noteStruct.glyph] || noteStruct.glyph;
+    }
+    if (noteStruct.font) {
+      this.setFont(noteStruct.font);
+    } else if (noteStruct.glyph === undefined) {
+      this.setFont(Tables.lookupMetricFontInfo('TextNote.text.fontSize'));
+    } else {
+      this.measureText();
+    }
+    // Scale the font size by 1/1.3.
+    const smallerFontSize = Font.convertSizeToPointValue(this.textFont.size) * 0.769231;
```

Comment:
Maybe someday we can make this 0.769231 customizable. For example, it could be in the Metrics class (aka CommonMetrics).